AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -60,33 +60,33 @@ - (void)testValidateEventName {
   NSString *validEventName = @"validEventName";
   NSString *shortEventName = @"e";
   NSString *eventName256 =
-      [NSString stringWithFormat:@"%@%@", @"_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_"
-                                          @"256_256_256_256_256_256_256_256_256_256_256_256_256_256_256",
-                                 @"_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_"
-                                 @"256_256_256_256_265_256_256_256_256_256_256"];
+  [NSString stringWithFormat:@"%@%@", @"_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_256_"
```

Comment:
The formatting here looks odd.

Reply:
[stringByPaddingToLength:withString:startingAtIndex:](https://developer.apple.com/documentation/foundation/nsstring/1416395-stringbypaddingtolength?language=objc)?